cl/citations/tests.py

```diff
@@ -761,3 +792,113 @@ def test_making_edge_list(self) -> None:
+                True,
+            ),
+            (
+                "where plaintif's complaint alleges facts which, if proven, would entitle plaintiff to relief under the Eighth Amendment, dismissal of complaint was inappropriate",
+                True,
+            ),
+            (
+                "ruling that there is nothing either legal or illegal, only thinking makes it so",
+                True,
+            ),
+            ("First Amendment", True),
+            ("mislabeled product", True),
+            ('"Look on my Works, ye Mighty, and despair"', True),
+            (
+                '"Texas does not seek to have the Court interpret the Constitution, so much as disregard it."',
+                True,
+            ),
+            ("questioning whether he who made the Lamb made thee", True),
+            ("Section 403(d)(2)", True),
+            ("Gonzales II", False),
+            ("Third Circuit 2013", False),
+            ("3d. Cir. 1776", False),
+            ("emphasis in original", False),
+            ("quotation altered", False),
+            ("internal citations and quotations omitted", False),
+            ("citations and internal ellipses omitted", False),
+            ("quotation marks omitted; ellipses ours", False),
+            ("plurality opinion", False),
+            ("opinion of Breyer, J.", False),
+            ("opinion of Mister Justice Black", False),
+            ("supplemental opinion", False),
+            ("Scalia, J., concurring in the judgment", False),
+            (
+                "Sotomayor, J., statement respecting denial of certiorari",
+                False,
+            ),
+            (
+                "Roberts, C.J., concurring in part and dissenting in part",
+                False,
+            ),
+            (
+                "Friendly, J., concurring in the judgment, concurring in part, and dissenting in part",
+                False,
+            ),
+            ("en banc", False),
+            ("per curiam", False),
+            ("dicta", False),
+            ("same", False),
+            ("no", False),
+            ("n. 3", False),
+            ("No. 12-345", False),
+            ("TILA", False),
+            ("citing Jones", False),
+            ("cited in Heart of Atlanta Motel v. United States", False),
+            (
+                "quoting Hart Steel Co. v. Railroad Supply Co., 244 U.S. 294, 299, 37 S. Ct. 506, 508, 61 L. Ed. 1148 (1917)",
+                False,
+            ),
+            ("collecting cases", False),
+        ]
+
+        for parenthetical_text, expected_is_descriptive in test_pairs:
+            self.assertEqual(
+                is_parenthetical_descriptive(parenthetical_text),
+                expected_is_descriptive,
+                f"Got incorrect result from is_parenthetical_descriptive for text (expected {expected_is_descriptive}): {parenthetical_text}",
+            )
```

Comment:
1. This and your following test would benefit from the use of self.subTest. 
2. I'd split up this test into two: test_is_good_description and test_not_not_good_description (or similar). That'd add some organization to your True/False test cases beyond putting the Falses latter, and it'd simplify your test variable so that it didn't have nested tuples.